script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
```

Comment:
**meters** Always use SI by default. NED at 1/3 arc second is ~10m thats great. Outside the domain, throw a warning and that querying elevation outside the US is not yet implemented, with a link to 'request this feature at https://github.com/pecanproject/bety/issues/new' unless there is an even easier alterinative (@yanliu-chn?)